csharp/src/Ice/SSL/TransceiverI.cs

```diff
@@ -462,17 +466,28 @@ private bool validationCallback(
             errors ^= (int)SslPolicyErrors.RemoteCertificateNotAvailable;
         }
 
+        if ((errors & (int)SslPolicyErrors.RemoteCertificateNameMismatch) > 0)
```

Comment:
Since it's bitmask logic, should this be != 0?

Reply:
We use this logic in other comparisons in this file. I would fix them.